### ISSUE TITLE
compiler warning with WIDE=Y and newtest

### ISSUE THREAD

**Issue by GitMensch** [Bill-Gray/PDCurses#58]
Environment:

~~~
$ uname -a
MINGW64_NT-10.0 MY-MACHINE 2.5.0(0.295/5/3) 2016-03-31 18:47 x86_64 Msys

$ cc --version
cc.exe (GCC) 4.8.1
~~~

Warnings:

~~~
../demos/newtest.c: In function 'dummy_main':
../demos/newtest.c:339:28: warning: passing argument 2 of 'swprintf' makes pointer from integer without a cast [enabled by default]
                            (wchar_t)( i + unicode_offset));
                            ^
In file included from ../demos/newtest.c:16:0:
c:\mingw\include\wchar.h:157:37: note: expected 'const wchar_t *' but argument is of type 'int'
 _CRTIMP int __cdecl __MINGW_NOTHROW swprintf (wchar_t*, const wchar_t*, ...);
~~~

Note: none of the other demos raise this warning.

**Comment by Bill-Gray** [comment]
I think I've just fixed this.  For some reason,  I don't see that warning (I'm running 5.3.1;  things may have changed.)  However,  this post: https://sourceforge.net/p/mingw/bugs/1728/ says that swprintf() has "issues" involving a disagreement between ISO-C and (surprise!) Microsoft.

I think you'd get it to compile if you did swprintf without the '20' argument in there.  However,  the simplest (and recommended) thing to do appears to be not to use swprintf() in a MinGW program.  This is the only place the function is used,  and _not_ using it is easy enough to arrange.  My latest commit does so.

**Comment by GitMensch** [comment]
Thank you for 40b419a4f427ba60f07c84c036825a6803ded548, I'll check if this fixes both the warning and the chrash #60 and report back.

**Comment by GitMensch** [comment]
Fixed with 40b419a.
